src/main/resources/db/migration/0250/V298__ApplicationCountryCode.sql

```diff
@@ -0,0 +1,9 @@
+-- Add country_code column
+ALTER TABLE accelerator.applications ADD country_code TEXT REFERENCES countries (code);
+
+-- Migrate existing columns by looking up first 3 of internal name
+UPDATE accelerator.applications
+SET country_code = countries.code
+FROM countries
+WHERE SUBSTR(accelerator.applications.internal_name, 1, 3) = countries.code_alpha3
+  AND SUBSTR(accelerator.applications.internal_name, 1, 3) != 'XXX';
```

Comment:
Out of curiosity, what happens if you leave off the XXX condition? I think it would end up setting the value to null since the query wouldn't find a countries row.

Having the condition is fine for clarity, of course.